cms/src/db/repositories/baseRepository.spec.ts

```diff
@@ -41,4 +41,126 @@ describe("baseRepository", () => {
         expect(result[0]._id).toBe(mockEnglishContentDto._id);
         expect(result[1]._id).toBe(mockFrenchContentDto._id);
     });
+
+    it("can find an item based on parent id", async () => {
+        const changeDoc = {
+            _id: "change1",
+            type: "change",
+            parentId: mockEnglishContentDto.parentId,
+        } as any;
+        db.docs.bulkPut([mockEnglishContentDto, changeDoc]);
+        const repository = new BaseRepository();
+
+        const result = await repository
+            .whereParentId(mockEnglishContentDto.parentId, DocType.Content)
+            .toArray();
+
+        expect(result.length).toBe(1);
+        expect(result[0]._id).toBe(mockEnglishContentDto._id); // Change should be excluded
+    });
```

Comment:
Since there's a bit of duplication going on, remember to port this to the app as well.